compiler/fuzzer/src/runner.rs

```diff
@@ -52,13 +52,15 @@ impl RunResult {
 impl<L: Borrow<Lir>> Runner<L> {
     pub fn new(lir: L, closure: Closure, input: Input) -> Self {
         let (mut heap, constant_mapping) = lir.borrow().constant_heap.clone();
+        let closure = closure.clone_to_heap(&mut heap).try_into().unwrap();
+        let arguments = input.arguments.clone_to_heap(&mut heap);
```

Comment:
These could share an address mapping. I'm even thinking about always requiring one so that we can't forget this

Reply:
True. I think always requiring a mapping would also be nice, but maybe in a future PR?